core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -129,10 +130,11 @@ <h4 translate="I18N_PLAYER_SHARE_THIS_EXPLORATION" class="share-exploration-foot
     line-height: 0.6;
     padding-left: 4px;
     text-overflow: clip;
+    text-align: center;
```

Comment:
This aligns the 'author profiles' text to the center with respect to parent div.  
![aaaa3](https://cloud.githubusercontent.com/assets/14854536/21728740/cb14a87a-d46d-11e6-953b-41e16cee62b7.png)